src/lambertw.jl

```diff
@@ -0,0 +1,386 @@
+import Base: convert
+#export lambertw, lambertwbp
+using Compat
+
+const euler =
+if isdefined(Base, :MathConstants)
+    Base.MathConstants.e
+else
+    e
+end
+
+const omega_const_bf_ = Ref{BigFloat}()
+
+function __init__()
+    omega_const_bf_[] =
+        parse(BigFloat,"0.5671432904097838729999686622103555497538157871865125081351310792230457930866845666932194")
+end
+
+#### Lambert W function ####
+
+const LAMBERTW_USE_NAN = false
```

Comment:
I must say, I don't really like this. I think we should pick a convention that matches the other functions here and stick to it.

Reply:
Ok. IIRC, there was some debate around the time I wrote the module about how to handle this in general.  I also wanted to see which I liked better. I think the issue is settled, so sure, I'll try to find the convention. It can only be changed by editing the source. In SpecialFunctions  this certainly shouldn't be advertised as something to do.